source/scss/components/_link-list.scss

```diff
@@ -34,11 +42,21 @@
 @include media-breakpoint-up("md") {
   .link-list {
     flex-wrap: nowrap;
+
+    &.single-title {
+      h3 {
+        padding: 0 $grid-gutter-width/2;
```

Comment:
Talked to Birgit and we agreed to leave this as this is, since there were more occurrences of this variable in this file before this commit (and it's better than just replacing them with '15px').